WordPress/src/main/java/org/wordpress/android/ui/compose/theme/AppThemeWihoutBackground.kt

```diff
@@ -11,22 +11,22 @@ import org.wordpress.android.BuildConfig
  * See [Compose Docs](https://developer.android.com/jetpack/compose/themes/material#content-color) for more details.
  */
 @Composable
-fun AppThemeWithBackground(
+fun AppTheme(
     isDarkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
-    AppTheme(isDarkTheme) {
+    AppThemeWithoutBackground(isDarkTheme) {
```

Comment:
Nice 👍 . I like the semantics of this when leaving public the theme without surface. Also, the doc comments are informative!

Reply:
Team work 🎸 🚀